### PR TITLE
Post type dropdown 767

### LINKED ISSUE #767: Tweak post creation modal post type choice
https://www.figma.com/file/BviY3FipADBW6ACBMIFSPt/Post-creation-type-dropdown?node-id=0%3A1&viewport=746%2C489%2C0.7371163964271545

Switch to a dropdown to change post type so it can also choose Project and Event

Create Project and Create Event buttons now go to the same modal with that type selected in the dropdown

### PR DESCRIPTION
**Pull request by cpolitano**
- Replaces post type button choices with a dropdown that includes all post types
- clicking FeedBanner from anywhere opens this same modal with the correct post type selected (including Events and Projects) 
- Closes #767

### CODE REVIEW

File: src/components/FeedBanner/FeedBanner.js [hunk 1]
```diff
@@ -31,7 +33,7 @@ export default function FeedBanner ({
   } else if (!group) {
     return null
   } else {
-    ({ bannerUrl, avatarUrl, name, location } = group)
+    ({ bannerUrl, avatarUrl, name, groupLocation } = group)
```
Comment (tibetsprague) [inline src/components/FeedBanner/FeedBanner.js:36]: this seems off to me, i know groups have a location, its not called groupLocation, does this work still?

**Review by tibetsprague — COMMENTED** [review]
looks good, just one question

**Comment by tibetsprague** [comment]
there is a failing snapshot but i think its because the display of the group location broke

**Comment by tibetsprague** [comment]
also theres now a merge conflict with the latest holonic-architecture

**Comment by tibetsprague** [comment]
one other thing i noticed thats a bit weird is when you change the post type it also filters (and reloads) the stream by that post type. darn. maybe we just change the query param to be a different one that the one used to filter the stream? it could be newPostType= ?

File: src/components/FeedBanner/FeedBanner.js (hunk 1 above)
Comment (cpolitano) [reply src/components/FeedBanner/FeedBanner.js:36]: mmm super good point

**Comment by cpolitano** [comment]
all set now!

**Review by tibetsprague — CHANGES_REQUESTED** [review]
please fix test then good!

File: src/components/CreateModal/CreateModal.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ import CreateGroup from 'components/CreateGroup'
 import CreateModalChooser from './CreateModalChooser'
 import Icon from 'components/Icon'
 import PostEditor from 'components/PostEditor'
+import Icon from 'components/Icon'
```
Comment (tibetsprague) [inline src/components/CreateModal/CreateModal.js:9]: this is whats breaking the tests

**Review by tibetsprague — APPROVED** [review]

**Pull request merged**